File: backend/src/BookSearch.DataAccess.Database.Entities/Book.cs
namespace BookSearch.DataAccess.Database.Entities
{
    public class Book : BaseEntity
    {
        public string? GoogleId { get; set; }

        public string? Title { get; set; }

        public string? Subtitle { get; set; }

        public IEnumerable<Author>? Authors { get; set; }

        public string? PublishedDate { get; set; }

        public string? Description { get; set; }

        public int PageCount { get; set; }

        public string? SmallThumbnail { get; set; }

        public string? Thumbnail { get; set; }

        public string? Language { get; set; }
    }
}

File: frontend/src/src/app/models/book.ts
import { Author } from "./author";

export interface Book {

    googleId: string;
    title: string;
    subtitle: string;
    authors: Author[];
    publishedDate: string;
    description: string;
    pageCount: number;
    smallThumbnail: string;
    thumbnail: string;
    language: string;
}